Sources/SwiftIfConfig/SyntaxLiteralUtils.swift

```diff
@@ -36,13 +36,23 @@ extension LabeledExprListSyntax {
 extension ExprSyntax {
   /// Whether this is a simple identifier expression and, if so, what the identifier string is.
   var simpleIdentifierExpr: String? {
-    guard let identExpr = self.as(DeclReferenceExprSyntax.self),
-      identExpr.argumentNames == nil
-    else {
+    guard let identExpr = self.as(DeclReferenceExprSyntax.self) else {
       return nil
     }
 
-    // FIXME: Handle escaping here.
-    return identExpr.baseName.text
+    return identExpr.simpleIdentifier
+  }
+}
+
+extension DeclReferenceExprSyntax {
+  /// If this declaration reference is a simple identifier, return that
+  /// string.
+  var simpleIdentifier: String? {
+    guard argumentNames == nil else {
+      return nil
+    }
+
+    /// FIXME: Make this an Identifier so we handle escaping properly.
```

Comment:
Any reason to not make it an `Identifier` now?

Reply:
Nope, we can do it now: https://github.com/swiftlang/swift-syntax/pull/2806